server/repository/src/tests.rs

```diff
@@ -287,27 +287,17 @@ mod repository_test {
         ItemLinkRowRepository, ItemRow, ItemRowRepository, KeyValueStoreRepository, KeyValueType,
         MasterListFilter, MasterListLineFilter, MasterListLineRepository,
         MasterListLineRowRepository, MasterListNameJoinRepository, MasterListRepository,
-        MasterListRowRepository, NameLinkRowRepository, NameRow, NameRowRepository,
-        NumberRowRepository, NumberRowType, RequisitionFilter, RequisitionLineFilter,
-        RequisitionLineRepository, RequisitionLineRowRepository, RequisitionRepository,
-        RequisitionRowRepository, StockLineFilter, StockLineRepository, StockLineRowRepository,
-        StorageConnection, StoreRowRepository, UserAccountRowRepository,
+        MasterListRowRepository, NameLinkRowRepository, NameRowRepository, NumberRowRepository,
+        NumberRowType, RequisitionFilter, RequisitionLineFilter, RequisitionLineRepository,
+        RequisitionLineRowRepository, RequisitionRepository, RequisitionRowRepository,
+        StockLineFilter, StockLineRepository, StockLineRowRepository, StorageConnection,
+        StoreRowRepository, UserAccountRowRepository,
     };
     use crate::{DateFilter, EqualFilter, StringFilter};
     use chrono::Duration;
     use diesel::{sql_query, sql_types::Text, RunQueryDsl};
     use util::inline_edit;
 
-    async fn insert_name_and_link(name: &NameRow, connection: &StorageConnection) -> () {
```

Comment:
Removed since new helper function in name_row does this anyway

Reply:
Good call! I should probably do the same for `insert_item_and_link`